battery.sh

```diff
@@ -406,6 +410,12 @@ fi
 
 # Maintain at level
 if [[ "$action" == "maintain_synchronous" ]]; then
+	
+	# Checking if the calibration process is running
+	if test -f "$calibrate_pidfile"; then
+		pid=$(cat "$calibrate_pidfile" 2>/dev/null)
+		log "🚨 Calibration is running, please run 'battery calibrate stop' or wait for it to finish"
```

Comment:
This line should probably not only log an error but exit the command so the processes do not conflict.

Reply:
You say , it should stop the battery calibration process?